portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,13 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Jaden Deristel's portfolio</h1>
+      <p>Hi! My name is Jaden Deristel and this is my webpage.</p>
+      <img src="/Images/image0.jpeg" alt="Image of Jaden" width="240" height="140">
```

Comment:
Nit: Using CSS for image sizing will make it easier to reuse.

Reply:
Noted, thank you. I will make those changes.